packages/@aws-cdk/aws-apigatewayv2/lib/websocket/route.ts

```diff
@@ -37,6 +37,13 @@ export interface WebSocketRouteOptions {
    * @default - No Authorizer
    */
   readonly authorizer?: IWebSocketRouteAuthorizer;
+
+  /**
+   * Should the route send a response to the client
+   * @default false
+   */
+  readonly returnResponse?: boolean;
```

Comment:
Given that the way this is being used when it is true, I'm not sure that boolean is the right value here. You're adding `$default` as the routeResponseSelectionExpression creating a whole object from this . Can you help me understand why a boolean here is the right user experience?

Reply:
Today the only valid value o the routeResponseSelectionExpression is `$default` or `undefined`. So to not overcomplicate it by passing a string I abstracted it to be a boolean instead.


See the quote below from https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-api-selection-expressions.html
> Evaluation of the route response selection expression produces a route response key. Eventually, this key will be used to choose from one of the [RouteResponses](https://docs.aws.amazon.com/apigatewayv2/latest/api-reference/apis-apiid-routes-routeid-routeresponses.html) associated with the API. However, currently only the $default key is supported.

But if we want to be more future proof we could change the attribute to be the actual `routeResponseSelectionExpression` and pass it a string.